spec/traits.dd

```diff
@@ -1349,6 +1352,12 @@ void test(alias pred)()
 
 void main()
 {
+
+    // similarly to `is(T1 == T2)`, types with qualifiers and suffixes
+    static assert(__traits(isSame, immutable(char)[], string));
+    static assert(!__traits(isSame, const(char)[], string));
+
+    // lambdas, common cases
```

Comment:
Can you move this example immediately after the new paragraph, as this example is already getting big?

Reply:
I'll see but I'm afraid that this could break the paragraph numbers.

Reply:
There's an example between 1. and 2., so in theory at least it should work. But if it turns out that it's too much work, then no need to bother.

Reply:
No this is definitively not too much work. I probably mismatch with another problem I've seen once while writing ddoc. I think that the issue was rather when you put code within a single paragraph.